jetty-server/src/main/java/org/eclipse/jetty/server/handler/FileBufferedResponseHandler.java

```diff
@@ -225,6 +220,14 @@ protected void onCompleteFailure(Throwable cause)
                     dispose();
                     callback.failed(cause);
                 }
+
+                @Override
+                public String toString()
+                {
+                    return String.format("FileBufferedInterceptor@%X[fileLength=%d,pos=%d,last=%b]",
```

Comment:
Don't hardcode the class name, use `getClass().getSimpleName()`.
Also, the hashCode should be `%x`, not `%X`.

Reply:
This is an anonymous class.

Reply:
I'll just remove this one, as we moved the register from this anonymous class to the outer interceptor anyway.